### ISSUE TITLE
Difficulty Overfitting Framepack on Small Datasets

### ISSUE THREAD

**Issue by WUyinwei-hah** [kohya-ss/musubi-tuner#432]
**Description:**
I encountered an issue when using Framepack for motion fitting experiments — it seems difficult to overfit on small datasets.

**Details:**

On a dataset containing only 10 videos, I tried an overfitting experiment by training for 80 epochs. However, the generated videos still showed a large gap compared to the reference videos.

When I further reduced the dataset to only 1 video, it still could not achieve overfitting (the generated results were still very different from the reference).

Intuitively, with datasets this small, the model should be able to overfit easily, but that’s not happening in my experiments.

**I’d like to ask:**

Has anyone else encountered the same issue?

Could this be related to the model architecture, training configuration, or data preprocessing?

Below are two example comparisons (reference vs. generated):

Ref.:
https://github.com/user-attachments/assets/a1b5976e-b570-43cf-b594-aa319e7a454a
Generated:
https://github.com/user-attachments/assets/8a7bd53c-5f9d-40b5-90ba-704ce67e1e0b

**Comment by arledesma** [comment]
FramePack Original or F1?

Optimizer?
Learning Rate?
What is your dataset configuration?
What does your captioning look like?
What does your loss look like?

The [discussions section](https://github.com/kohya-ss/musubi-tuner/discussions) is probably be a better place for this. There are some [FramePack specific discussions](https://github.com/kohya-ss/musubi-tuner/discussions?discussions_q=is%3Aopen+framepack) that may point you in the correct direction, but it's hard to say without knowing exactly what you are attempting to overfit.

**Comment by WUyinwei-hah** [comment]
Thanks for the quick reply! Here are my answers to your questions:

FramePack Version: Tried both Original and F1.

Optimizer: adamw8bit with learning rate 1e-4.

Dataset Configuration:

Initially used a dataset with ~40k samples. After training, the generated results showed almost no motion change — only appearance changes.

To test overfitting, I reduced the dataset to 10 videos and even 1 video, but the results were still unsatisfactory, with the model unable to overfit even in the 1-video case.

Captioning: Mainly short captions with only essential information, e.g.:

> The green dinosaur moves around and stomps its feet.

I can also share the loss curves if that would help with further diagnosis.

<img width="1229" height="434" alt="Image" src="https://github.com/user-attachments/assets/5cac5ed1-7551-46a2-8721-ddc4daaa5c42" />
